### PR TITLE
Problem: Every dependency is a risk to consumers of cyverse-ui

### PR DESCRIPTION
**Pull request by Tharon-C**
This resolves [ATMO-1837](https://pods.iplantcollaborative.org/jira/browse/ATMO-1837)

Because of a temporary bug in `react-scroll` fresh builds were failing for a full day. This PR reduces the amount of dependencies as an attempt to reduce the potential points of failure. 

- Remove unused dependencies
- Replace `radium` with own HOC "hoverable"
We were only using Radium as a more convenient way to style hover state, this is simple enough in a HOC 
- Remove `react-collapse`, doesn't add enough value to MediaCards
After removing `react-collapse` and testing the height animation didn't add enough value as the animated scroll, margins, and width gave enough motion to give the user the context necessary to understand the action of the card expanding.

### CODE REVIEW

File: guide/src/examples/TabsEx.js [hunk 1]
```diff
@@ -20,31 +21,43 @@ export default React.createClass({
...
-                    tabList={[
-                        "Red Fish",
-                        "Blue Fish",
-                        "Green Fish",
-                    ]}
-                    current={this.state.currentTab}
-                    color={ theme.color.primary }
-                    onTabClick={ this.onTabClick }
-                />
+                <Paper
```
Comment (Tharon-C) [inline guide/src/examples/TabsEx.js:24]: While I was refactoring Tab styles I remembered we don't want to style the background as there are different contexts in which one might need to display it. In this case we display it on Paper.

Comment (lenards) [reply guide/src/examples/TabsEx.js:24]: Okay.

**Comment by lenards** [comment]
Let's good. 

I'll let Connor offer his feedback too.

**Comment by lenards** [comment]
Due to build failures on release branches of Troposphere, I'm merging this:

Example ❌  https://travis-ci.org/cyverse/troposphere/jobs/224007424

**Comment by lenards** [comment]
:shipit:

**Comment by lenards** [comment]
Any feedback that one has on this, please make a _counter_ pull request if there are code changes. Otherwise, comment here and tag us via @ + handle.

**Pull request merged**